Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:7.3-cli
+
+WORKDIR /app
+
+RUN apt-get update && \
+	apt-get install -y libxml2-dev && \
+	docker-php-ext-install soap
+
+COPY . .
```

Comment:
Jaky je ucel tohoto ``COPY`` kdyz se v docker-compose mountuje volume?

Reply:
Kdyby někdo nepoužil docker-compose, případně kdyby někdo nepoužil mountování z docker-compose.

Reply:
V tom pripade bych pouzil multistage build aby se pro vyvoj a pousteni testu pred docker-compose nepouzival image ktery kopiruje data a pote misto nich mountuje slozku.

Napadlo mne ze by to slo postavit na [``fmasa/lebeda:7.3``](https://github.com/fmasa/dockerfiles/tree/master/lebeda) a vlastne tim testovat ze knihovna je kompatibilni s Lebedou. (Pridali bychom spusteni testu v  dockeru do Travisu).

Budes mit zajem to implementovat nebo se do toho mam pustit?

Reply:
Co by delal ten multistage build?

aklidne zmenim base image, to je asi drobnost...

Reply:
Jedna stage by pripravila environment (php, extensions) a na tu by se odkazoval docker-compose pomoci ```target``` vlastnosti pro ```build```. A druha stage by obsahovala ``copy``.